libs/api/core/data-access/src/prisma/schema.prisma

```diff
@@ -191,6 +194,7 @@ model Cluster {
   updatedAt   DateTime      @default(now()) @updatedAt
   enableStats Boolean       @default(false)
   endpoint    String
+  explorer    String
```

Comment:
After doing `yarn prisma db push` I am getting 

```
We found changes that cannot be executed:

  • Added the required column `explorer` to the `Cluster` table without a default value. There are 2 rows in this table, it is not possible to execute this step.
  ```

And it doesn't allow me to do the push. Maybe do we need to set a default value?

Reply:
The value is set during the provisioning of the clusters on startup so we're fine with not having a default in the db.